recipe/meta.yaml

```diff
@@ -21,11 +22,11 @@ requirements:
     - setuptools
   run:
     - python >=3.4
```

Comment:
Please increase this to 3.9, just because it is in the `setup.py` doesn't mean it is correct 😛